src/CMakeLists.txt

```diff
@@ -759,9 +771,16 @@ ELSE(APPLE OR WIN32)
 ENDIF(APPLE OR WIN32)
 
 OPTION(VISIT_THREAD "Build VisIt with thread support." OFF)
-OPTION(VISIT_BUILD_AVTEXAMPLES  "Enable building of avtexamples" ON)
-OPTION(VISIT_BUILD_DIAGNOSTICS  "Enable building of diagnostices" ON)
 
+# some options that when turned off reduce the build targets
+OPTION(VISIT_ENABLE_ANNOTATION_TOOLS "Enable generation of annotation tools (text2polys, time_annotation)" ON)
+OPTION(VISIT_ENABLE_DATAGEN "Enable generation of sample data files" ON)
+OPTION(VISIT_ENABLE_DATA_MANUAL_EXAMPLES "Build Getting Data Into VisIt examples" OFF)
+OPTION(VISIT_ENABLE_DIAGNOSTICS  "Enable building of diagnostices" ON)
+OPTION(VISIT_ENABLE_EXAMPLES  "Enable building of examples" ON)
+OPTION(VISIT_ENABLE_LIBSIM "Enable generation libsim." ON)
```

Comment:
Is `of` missing between `generation` and `libsim`? What is rational for `generation` vs `building`? Does `generation` actually create data or does it just build the executables which, if run, create data? Probably best to use `building` everywhere consistently here.

Reply:
Good catch. I fixed per your suggestion.